include/effects.h

```diff
@@ -1797,18 +1797,18 @@ typedef struct HieroglyphsFXData {
 } HieroglyphsFXData; // size = 0x3C
 
 typedef struct MiscParticlesFXData {
-    char unk_00[0x4];
-    Vec3f pos;
-    f32 scaleX;
-    f32 scaleY;
-    char unk_18[0x20];
-    s32 unk_38;
-    s32 unk_3C;
-    s32 unk_40;
-    char unk_44[0x4];
-    s32 unk_48;
-    s32 unk_4C;
-    s32 unk_50;
+    /* 0x00 */ char unk_00[0x4];
+    /* 0x04 */ Vec3f pos;
+    /* 0x10 */ f32 scaleX;
+    /* 0x14 */ f32 scaleY;
+    /* 0x18 */ char unk_18[0x20];
+    /* 0x38 */ s32 unk_38;
+    /* 0x3C */ s32 unk_3C;
+    /* 0x40 */ s32 unk_40;
+    /* 0x44 */ char unk_44[0x4];
+    /* 0x48 */ s32 unk_48;
+    /* 0x4C */ s32 unk_4C;
+    /* 0x50 */ s32 unk_50;
 } MiscParticlesFXData; // size = ?
```

Comment:
```suggestion
} MiscParticlesFXData; // size = 0x54
```

Reply:
We don't know the size of this yet, as it's not decompiled

Reply:
Ignore :)

Reply:
nah I just saw that it was missing comments too